packages/core/src/App/Containers/server-time.jsx

```diff
@@ -3,18 +3,21 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { toGMTFormat } from '@deriv/shared';
 import { connect } from 'Stores/connect';
+import { Popover } from '@deriv/components';
 
 const ServerTime = ({ is_mobile, server_time }) => {
     const gmt_time = toGMTFormat(server_time);
 
     return (
-        <div
+        <Popover
+            alignment='top'
+            message={gmt_time}
```

Comment:
We should show local time when hovered just like `binary.com`

Reply:
I see, I will fix that